lib/store/index.js

```diff
@@ -1,20 +1,21 @@
-import actions from '~typo3/store/actions'
-import mutations from '~typo3/store/mutations'
+import { state, actions, mutations } from '~typo3/store/typo3.js'
 
 export default ({ store }) => {
+  let moduleOptions = <%= serialize(options) %>
+
   store.registerModule('typo3', {
```

Comment:
Seems like module registration fails,

There are a lot of errors in the browser console. The interesting fact is that the page is rendered successful after all.
```
app.js:1855 [vuex] unknown mutation type: typo3/SET_LOCALE_ACTIVE
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 [vuex] unknown action type: typo3/nuxtServerInit
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 [vuex] unknown action type: typo3/getInitialData
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 TypeError: Cannot read property 'then' of undefined
    at :3000/server.js:8336:11
    at new Promise (<anonymous>)
    at setLocale (:3000/server.js:8331:10)
    at Object.setLocale (:3000/server.js:8814:116)
    at module.exports../example/.nuxt/typo3/middleware.i18n.js._middleware__WEBPACK_IMPORTED_MODULE_0__.default.typo3i18n (:3000/server.js:3769:21)
    at promisify (:3000/server.js:4355:15)
    at middlewareSeries (:3000/server.js:4334:10)
    at :3000/server.js:4335:12
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (:3000/internal/process/task_queues.js:95:5)
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 [vuex] unknown action type: typo3/getInitialData
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 TypeError: Cannot read property 'then' of undefined
    at :3000/server.js:8336:11
    at new Promise (<anonymous>)
    at setLocale (:3000/server.js:8331:10)
    at Object.setLocale (:3000/server.js:8814:116)
    at module.exports../example/.nuxt/typo3/middleware.i18n.js._middleware__WEBPACK_IMPORTED_MODULE_0__.default.typo3i18n (:3000/server.js:3769:21)
    at promisify (:3000/server.js:4355:15)
    at middlewareSeries (:3000/server.js:4334:10)
    at :3000/server.js:4335:12
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (:3000/internal/process/task_queues.js:95:5)
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 [vuex] unknown mutation type: typo3/SET_PAGE_DATA
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
app.js:1855 [vuex] module namespace not found in mapState(): typo3/
(anonymous) @ client.js:57
(anonymous) @ client.js:57
./example/.nuxt/client.js @ app.js:1855
__webpack_require__ @ bootstrap:853
fn @ bootstrap:150
0 @ app.js:18228
__webpack_require__ @ bootstrap:853
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ app.js:1
Show 7 more frames
vue.runtime.esm.js:8484 You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html
client.js:95 [HMR] connected
default.vue?4bcd:9 [vuex] module namespace not found in mapState(): typo3/
```

Reply:
Ok I see, I have forgotten about the namespaced property